lms/djangoapps/edraak_university/helpers.py

```diff
@@ -0,0 +1,80 @@
+"""
+Helpers for the University ID django app, mainly for correct feature checks.
+"""
+
+from django.contrib.auth.models import AnonymousUser
+from django.conf import settings
+
+from opaque_keys.edx.locator import CourseLocator
+
+from edraak_university.models import UniversityID
+
+
+def is_feature_enabled():
+    """
+    Checks if the feature is enabled, useful to avoid writing the feature flag everywhere.
+    """
+    return settings.FEATURES.get('EDRAAK_UNIVERSITY_APP')
+
+
+def is_csv_export_enabled_on_course(course):
+    """
+    Checks if the CSV grade report should include the University ID fields on a given course.
+    """
+    export_feature_enabled = bool(settings.FEATURES.get('EDRAAK_UNIVERSITY_CSV_EXPORT'))
+
+    return is_feature_enabled() and export_feature_enabled and course.enable_university_id
+
+
+def is_feature_enabled_on_course(course, user=None):
+    """
+    Correctly checks if the feature is enabled on a given course.
+    """
+    if settings.ROOT_URLCONF == 'lms.urls' and not user:
+        # For some reasons CMS provides no user
+        return False
+
+    if is_feature_enabled():
+        if course.enable_university_id:
+            return True
```

Comment:
It won't. The course object is not a dictionary. The field always exists regardless of the feature.